vcpkg-artifacts/tsconfig.json

```diff
@@ -9,6 +10,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitThis": true,
     "inlineSourceMap": true,
+    "sourceRoot": "https://github.com/microsoft/vcpkg-tool/tree/main/vcpkg-artifacts",
```

Comment:
Should this be a raw link?

`https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/vcpkg-artifacts`

Reply:
:shrug: